sfge/src/main/java/com/salesforce/graph/build/InheritanceEdgeBuilder.java

```diff
@@ -108,36 +121,40 @@ private Optional<InheritableSFVertex> findExtendedVertex(InheritableSFVertex ver
         }
     }
 
-    private List<InheritableSFVertex> findImplementedVertices(InheritableSFVertex vertex) {
-        if (vertex instanceof UserInterfaceVertex) {
-            return new ArrayList<>();
-        } else {
+    private Map<String, InheritableSFVertex> findImplementedVerticesByName(
```

Comment:
This returns a Map instead of a List, and interface names that don't have an associated vertex will be mapped to `null`.